packages/api/src/utils/date-helpers.ts

```diff
@@ -15,15 +17,53 @@ export const dateHelpers = {
   },
 
   parseGoogleDate(
-    googleDate: { date?: string; dateTime?: string },
+    googleDate: { date?: string; dateTime?: string; timeZone?: string },
```

Comment:
Nice! Adding the timeZone field everywhere saves us time transitioning to Temporal later.